MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -436,6 +438,19 @@ - (IBAction)onClearCacheButtonTapped:(id)sender
     }
 }
 
+- (BOOL)clearAllTokenKeys
+{
+    NSDictionary *query = @{(__bridge id)kSecClass: (__bridge id)kSecClassKey};
```

Comment:
This might impact WPJ state as well. Can we pass keychain group here?

Reply:
Updated code to instead use MSIDAssymetricKeyKeychainGenerator to delete the key which internally adds the access group to the query by default.